config/routes.rb

```diff
@@ -8,4 +8,6 @@
   resources :events, only: %i[index show new create]
 
   resources :attendances, only: %i[create destroy], param: :event_id
+
+  resources :users, only: %i[show]
```

Comment:
this is fine but just so you know, sometimes I've seen this getting sorted in a different way. You can create a [single resource](https://guides.rubyonrails.org/routing.html#singular-resources) called `profile` 

```ruby
resource :profile, only: %i[show]
```

that way the request that is handled by `#show` is `GET /profile`. You don't need to pass an ID on the URL

Reply:
nice! done!